critiquebrainz/frontend/static/scripts/rating.js

```diff
@@ -0,0 +1,157 @@
+(function ($) {
```

Comment:
Yes, I decided for this plugin after a lot of searching and still didn't find any other which is more suitable. I have updated the package file and working on the required changes. Will let you know at the earliest if I face any problems.